### PR TITLE
makes horizon docking ports functional 2: the dockening: rise of the orion ship

### PR DESCRIPTION
**Pull request by anconfuzedrock**
This modifies one of the horizon's docking ports, namely the one used by the orion ship, to actually function for docking, giving it the same frequency as the orion ship. This also fixes the orion ship's docking controller not functioning for docking. This allows it to dock with the horizon's fixed docking port. Finally, this fixes the orion ship's aft docking port and adds a transit marker to it, allowing other ships to dock with the orion ship if their entrance is north facing.

edit: additionally, this moves the transit area of the mining shuttle up so that it itself can dock with docking ports properly now, and it sets the airlock and shuttle I recently moved around to the same frequency as the common general ship docking frequency to avoid conflicts in the future.

### CODE REVIEW

**Comment by anconfuzedrock** [comment]
!review

**Comment by anconfuzedrock** [comment]
!review

**Comment by anconfuzedrock** [comment]
!wip

**Comment by anconfuzedrock** [comment]
!review

File: html/changelogs/anconfuzedrock-oriondock.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+author: anconfuzedrock
+
+delete-after: True
+
+changes:
+  - tweak: "the horizon's docking port that's used by the orion express ship now shares a frequency with it, allowing it to actually dock."
```
Comment (SleepyGemmy) [inline html/changelogs/anconfuzedrock-oriondock.yml:6]:
```suggestion
  - tweak: "The Horizon's docking port that's used by the Orion express ship now shares a frequency with it, allowing it to actually dock."
```

File: html/changelogs/anconfuzedrock-oriondock.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  - tweak: "The mining pod transit marks have been moved, so they can now dock properly with any functional south-facing dock."
```
Comment (SleepyGemmy) [inline html/changelogs/anconfuzedrock-oriondock.yml:7]:
```suggestion
  - tweak: "The Spark's transit marks have been moved, so they can now dock properly with any functional south-facing dock."
```

**Review by SleepyGemmy — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Comment by anconfuzedrock** [comment]
f it, time to fix this the cool way

**Pull request closed without merging**